packages/x-components/src/x-modules/history-queries/components/history-query.vue

```diff
@@ -21,6 +21,7 @@
       class="x-history-query__remove"
       :historyQuery="suggestion"
       data-test="remove-history-query"
+      aria-label="remove"
```

Comment:
I'm wondering whether this `aria-label` should go here or directly inside `remove-history-query.vue` 🤔 How do you make that decission? (Sorry if I'm missing something, I don't have any expierence on this topic so first I would like to understand how do you face this kind of issues).

Reply:
Due to the HistoryQuery component structure I would say there's no difference between putting it inside `remove-history-query.vue` or `history-query.vue` but yeah, you're right, I'll change it even if it is just for consistency purposes.